packages/server-wallet/src/chain-service/chain-service.ts

```diff
@@ -163,24 +209,30 @@ export class ChainService implements ChainServiceInterface {
 
     const captureExpectedErrors = async (reason: any) => {
       if (reason.error?.message.includes('Channel finalized')) {
+        this.logger.warn(
+          {channelId, determinedBy: 'Revert reason'},
```

Comment:
We might want to include `reason.error.message` in the log?

Reply:
The reason is that the channel is already finalized which is in the message; did you mean verbatim?

Reply:
I guess the only thing we could find out by including `reason.error.message` is whether the error is coming from transaction submission (in which case we will see a "failed to estimate gas" error) or transaction mining.

Reply:
In general I think the `ChainService` should hide the gas estimation wrapped errors from the consumer — the actual error is the interesting bit that the consumer wants to know about, it just happens to be that the RPC call to estimate gas consumption provides it to the `ChainService` to read